src/_includes/graphql/customer-orders-output.md

```diff
@@ -286,14 +355,24 @@ Attribute | Data type | Description
 
 Attribute | Data type | Description
 --- | --- | ---
-`id` | String! | The name of the option
+`label` | String! | The name of the option
 `value` | String! | The value of the option
 
+#### OrderPaymentMethod attributes {#OrderPaymentMethod}
+
+The OrderPaymentMethod data type contains details about the payment method used to pay for the order.
+
+Attribute | Data type | Description
+--- | --- | ---
+`additional_data` | [[KeyValue](#KeyValue)] | Additional data per payment method type
+`name` | String! | The label that describes the payment method
+`type` | String! | The payment method code that indicates how the order was paid for
```

Comment:
can we/should we list the possible payment method codes that this attribute can return?

Reply:
****